common/models.py

```diff
@@ -155,16 +189,16 @@ def update_from_pubmed_data(self, index=None):
                 # Sometimes 'DA' field does not exist, use alternative
                 self.year = record['DP'][:4]
 
-            record['JT'] = record['JT']
-            record['TA'] = record['TA']
+            #record['JT'] = record['JT']
+            #record['TA'] = record['TA']
```

Comment:
Maybe remove this? What even was this in the first place?

Reply:
No clue, now removed it.